lazycatlabs/flutter_auth_app
# feat: update gradle config

## What it does

- Update gradle configuration following the [documentation](https://flutter.dev/go/flutter-gradle-plugin-apply) to resolve this warning message
```
You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/go/flutter-gradle-plugin-apply
``` 
- Set Firebase Options manually to resolve issues after updating the gradle configuration
- Update library version

## How to test

1. Clone the project
2. Run the project 
3. Observe the log

## Screenshot
n/a
